src/Toggle/Toggle.tsx

```diff
@@ -43,18 +43,18 @@ export interface ToggleProps {
  * It has a position prop to change the layout of the label. State for the value needs to be maintained in a parent
  * component and passed in as a prop.
  */
-const Toggle: FC<ToggleProps> = ({
+const Toggle = <T, >({
```

Comment:
The trailing comma here is to keep this from looking like a JSX tag, which would need a closing tag.